manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt

```diff
@@ -17,7 +17,7 @@ To use this box run:
 {linenos=off, lang=bash}
 ~~~~~~~
 cd $pve/ariadne/orchestration/vagrant
-vagrant init deb/jessie-amd64 --provider virtualbox
+vagrant init deb/jessie-amd64
 ~~~~~~~
```

Comment:
With the "--provider" option, it gives me an error (Vagrant 1.7.2) -  "An invalid option was specified. The help for this command is available below.". And VirtualBox is the default provider, so no need to set it explicitly - https://docs.vagrantup.com/v2/providers/default.html